src/components/VoteCard.jsx

```diff
@@ -12,19 +12,19 @@ export default function VoteCard({votes, article_id}) {
     updateArticleVote({inc_votes: -1});
   };
   const updateArticleVote = (vote) => {
+    setErrMsg(undefined);
+    setNewVotes((currVotes) => {
+      return currVotes + vote.inc_votes;
+    });
     patchArticleVotes(article_id, vote)
-      .then(({article}) => {
+      .then(({article}) => {})
```

Comment:
You can delete this line seem as though you're not handling any data, just have the request and the catch